shell/platform/linux/fl_engine.cc

```diff
@@ -571,8 +593,37 @@ void fl_engine_send_mouse_pointer_event(FlEngine* self,
   FlutterEngineSendPointerEvent(self->engine, &fl_event, 1);
 }
 
+gboolean fl_engine_mark_texture_frame_available(FlEngine* self,
+                                                int64_t texture_id) {
+  g_return_val_if_fail(FL_IS_ENGINE(self), FALSE);
+  return FlutterEngineMarkExternalTextureFrameAvailable(self->engine,
+                                                        texture_id) == kSuccess
+             ? TRUE
```

Comment:
`== kSuccess` is sufficient to return a boolean, i.e. don't need the `? TRUE : FALSE`. (Same below).